cmd/experimental/kjobctl/pkg/cmd/describe/describe_task.go

```diff
@@ -0,0 +1,193 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package describe
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/cli-runtime/pkg/genericiooptions"
+	"k8s.io/cli-runtime/pkg/resource"
+
+	"sigs.k8s.io/kueue/cmd/experimental/kjobctl/pkg/cmd/util"
+)
+
+const (
+	managedLabel = "kjobctl.x-k8s.io/managed"
```

Comment:
Maybe we can move it to constants file like [this](https://github.com/kubernetes-sigs/kueue/blob/707b96c54fcb7efc01632b6ed896fdf4f4dc6288/cmd/experimental/kjobctl/pkg/constants/constants.go)? To use it in another places too. WDYT?